workflows/data_pipelines/convcollective/task_functions.py

```diff
@@ -85,6 +93,12 @@ def send_file_to_minio():
                 "dest_path": "convention_collective/new/",
                 "dest_name": "cc.csv",
             },
+            {
+                "source_path": CC_TMP_FOLDER,
+                "source_name": "metadata.json",
+                "dest_path": "convention_collective/new/",
+                "dest_name": "metadata.json",
+            },
```

Comment:
I feel like a large part of this PR could be factorize with a helper : 
`get_minio_metadata_file(CC_TMP_FOLDER, "convention_collective/new/")`
that returns 
```
{
                "source_path": CC_TMP_FOLDER,
                "source_name": "metadata.json",
                "dest_path": "convention_collective/new/",
                "dest_name": "metadata.json",
            },
```